### DISCUSSION TITLE
Would you consider RedwoodJS Example repository?

### DISCUSSION THREAD

**Discussion by adriatic** [saadeghi/daisyui#1645] [Ideas / feature requests]
I am in the process of creating an article, describing the daisyui use in the context of RedwoodJS application development - here is my [placeholder](https://community.redwoodjs.com/t/how-to-use-redwoodjs-applications-with-several-ui-component-libraries/4650#daisyuihttpsdaisyuicom-7).

**Comment by saadeghi** [comment]
Thank you. I'm not familiar with Redwood but if you manage to setup Tailwind CSS, adding daisyUI will be a 5 sec job. I found [this post](https://mdv.io/tailwindcss-in-redwoodjs) from 2020 on how to add Tailwind to Redwood. I don't know If there is a newer or cleaner way to do it tho.

**Reply by adriatic** [reply]
Hello, @saadeghi - I will definitively follow your suggestion and have little doubt that I will succeed. 

[RedwoodJS](https://redwoodjs.com/) is the last project of a very successful entrepreneur best known as the creator of GitHub [Tom Preston-Werner](https://en.wikipedia.org/wiki/Tom_Preston-Werner).

**Reply by saadeghi** [reply]
@adriatic I found this one too. Here on Redwood docs mentioned about how to add Tailwind: https://redwoodjs.com/docs/webpack-configuration#sass-and-tailwind-css

**Reply by adriatic** [reply]
Thank you for this reference (I am aware of it 😄 ). Since not everyone reads all of the RedwoodJS online documentation I am preparing a series of community supported articles in [RedwoodJS Community Library](https://rw-community.org/), with more details than just a single command `yarn rw setup ui tailwindcss`.
